mtk.go

```diff
@@ -0,0 +1,32 @@
+package nmea
+
+import "strconv"
+
+const (
+	// TypeMTK type for PMTK sentences
+	TypeMTK = "PMTK"
+)
+
+// MTK is the Time, position, and fix related data of the receiver.
+type MTK struct {
+	BaseSentence
+	Cmd,
+	Flag int
+}
+
+// newMTK constructor
+func newMTK(s BaseSentence) (MTK, error) {
+	cmd, err := strconv.Atoi(s.Fields[0])
```

Comment:
Use the `parser` type so you get consistent error messages.

Reply:
But the parser has some methods that are not relevant to the MTK type like Time, Date, LatLong, SixBitASCIIArmour

Reply:
The methods cover everything needed by all possible messages. Just use the ones you need.

Reply:
Thanks , updated.